doc/en/user/source/styling/webadmin/index.rst

```diff
@@ -140,6 +140,12 @@ The style editor supports line numbering, automatic indentation, and real-time s
      - Redo
    * - .. image:: img/styles_editor_goto.png
      - Go to line
+   * - .. image:: img/styles_editor_find.png
+     - Find in style text (CTRL-F)
+   * - .. image:: img/styles_editor_find_next.png
+     - Find next occurrence in style text (CTRL-G)
+   * - .. image:: img/styles_editor_replace.png
+     - Find and replace in style text (CTRL-SHIFT-F)
    * - .. image:: img/styles_editor_reformat.png

```

Comment:
I found the actual operation of the replace function a bit unintuitive to use - I'm used to having "Replace" and "With" together in the same dialog. A doc note indicating that "Enter" lets you set the "With" is probably a good idea.